src/CqlBuilderPanel/codesSection/CodesSection.test.tsx

```diff
@@ -112,7 +112,7 @@ describe("CodesSection", () => {
     const savedCodesSectionButton = await screen.findAllByRole("button");
     const savedCodesSectionHeading = within(
       savedCodesSectionButton[0]
-    ).getByText("Saved Codes");
-    expect(savedCodesSectionHeading).toBeInTheDocument();
+    ).queryByText("Saved Codes");
+    expect(savedCodesSectionHeading).not.toBeInTheDocument();
```

Comment:
we probably do not need to check for the thing that is not part of the functionality anymore

Reply:
@adongare Yeah sure will remove. Thanks!

Reply:
@adongare I have made those 2 changes, thanks!